must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -107,7 +113,11 @@ for ns in $namespaces; do
     if [ "${operatorImage}" = "" ]; then
         echo "not able to find the rook's operator image. Skipping collection of ceph command output" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
     elif [[ $cephClusterCount -gt 0 ]]; then
-        apply_helper_pod "$ns" "$operatorImage"
+        if [[ oc get csv -n "${ns}" --no-headers | awk '{print $5}' -gt 4.5.0 ]]; then
+            apply_latest_helper_pod "$ns" "$operatorImage"
+        else
+            apply_helper_pod "$ns" "$operatorImage"
```

Comment:
just a naming suggestion and I feel it might be subjective but, instead of naming `apply_helper_pod`, change it to `apply_standard_helper_pod` since `latest` means when OCS version is > 4.5.0 and `standard` means the usual pod when the OCS version is <= 4.5.0.
What do you think?,

Reply:
yes naming has to be correct. How about we keep the latest for 4.5.0> and standard for 4.6.0<.

Reply:
Yes
That would make much more sense